index.html

```diff
@@ -1,102 +1,109 @@
-              fill="currentColor"
-            />
-          </svg>
-          Read me</a
-        >
-      </h3>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en-gb">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My Portfolio</title>
+    <meta
+      name="description"
+      content="The technical portfolio of trainee name"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!-- suppress FOUC-->
+    <style>
+      html {
+        animation: fade-in 1s;
+      }
+      @keyframes fade-in {
+        0% {
+          opacity: 0;
+        }
+        100% {
+          opacity: 1;
+        }
+      }
+    </style>
+    <link
+      rel="stylesheet"
+      href="style.css"
+      media="print"
+      onload="this.media='all'"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <h1>Bakhat Begum</h1>
+      <nav>
+        <ul>
+          <li><a href="#about">About Me</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main tabindex="0">
+      <section id="about">
+        <header><h2>About Me</h2></header>
+        <p>
+          I am currently studying software development bootcamp at
+          CodeYourFuture. I am a hardworking student who is committed to
+          pursuing my goals in becoming a software developer and advancing my
+          career in tech field.
+        </p>
+      </section>
+      <section id="projects">
+        <header><h2>Projects Showcase</h2></header>
+        <p>
+          Passenger-Account, Blackjack-game, Mobile-configure and
+          Basic-calculater is my own project that I have created during my
+          spare time.
+
+          <p>Link to my works:</p>
+
+          <p>
+          https://github.com/BakhatBegum/Blackjack-game<br />

```

Comment:
You want these links to be clickable! Not just text! How can you fix it? Have a read here! https://www.w3schools.com/html/html_links.asp